.jenkins/Jenkinsfile-wheel-builder

```diff
@@ -0,0 +1,150 @@
+                def pythonPath = [
+                    py26: "/usr/bin/python2.6",
+                    py27: "/Library/Frameworks/Python.framework/Versions/2.7/bin/python2.7",
+                    py34: "/Library/Frameworks/Python.framework/Versions/3.4/bin/python3.4",
+                    py35: "/Library/Frameworks/Python.framework/Versions/3.5/bin/python3.5",
+                    py36: "/Library/Frameworks/Python.framework/Versions/3.6/bin/python3.6",
+                ]
+                ansiColor {
+                    sh """#!/usr/bin/env bash
+                        set -xe
+                        # output the list of things we've installed as a point in time check of how up
+                        # to date the builder is
+                        /usr/sbin/system_profiler SPInstallHistoryDataType
+
+                        # Jenkins logs in as a non-interactive shell, so we don't even have /usr/local/bin in PATH
+                        export PATH="/usr/local/bin:\${PATH}"
+                        export PATH="/Users/jenkins/.pyenv/shims:\${PATH}"
+
+                        printenv
+
+                        virtualenv .venv -p ${pythonPath[version]}
+                        source .venv/bin/activate
+                        pip install -U wheel # upgrade wheel to latest before we use it to build the wheel
+                        pip wheel bcrypt --wheel-dir=wheelhouse --no-binary bcrypt
+                        pip install -f wheelhouse bcrypt --no-index
+                        python -c "import bcrypt;password = b'super secret password';hashed = bcrypt.hashpw(password, bcrypt.gensalt());assert hashed"
+                    """
+                }
+            } else if (label.contains("docker")) {
+                linux32 = ""
+                if (imageName.contains("i686")) {
+                    linux32 = "linux32"
+                }
+                sh """#!/usr/bin/env bash
+                    set -x -e
+                    # Because we are doing this as root in the container, but we write to a mounted dir that is outside the container
+                    # we need to make sure we set these files writable such that the jenkins user can delete them afterwards
+                    mkdir -p tmpwheelhouse
+                    mkdir -p wheelhouse
+                    chmod -R 777 tmpwheelhouse
+                    chmod -R 777 wheelhouse
+
+                    $linux32 /opt/python/$version/bin/pip install cffi six
+                    $linux32 /opt/python/$version/bin/pip wheel --no-binary bcrypt --no-deps bcrypt -w tmpwheelhouse/
+                    $linux32 auditwheel repair tmpwheelhouse/bcrypt*.whl -w wheelhouse/
+                    $linux32 /opt/python/$version/bin/pip install bcrypt --no-index -f wheelhouse/
+                    $linux32 /opt/python/$version/bin/python -c "import bcrypt;password = b'super secret password';hashed = bcrypt.hashpw(password, bcrypt.gensalt());assert hashed"
```

Comment:
Bad reasons (I'm pulling together code from within the old jenkins, the wheel scripts, and other repos and picked an older version of this verification code from jenkins rather than the better code that was already in the wheel-scripts)